usb/libloader.py

```diff
@@ -179,16 +181,20 @@ def load_locate_library(candidates, cygwin_lib, name,
         else:
             raise NoLibraryCandidatesException(name)
     elif candidates:
-        lib = locate_library(candidates, find_library)
-        if lib:
-            _LOGGER.debug("%r found as %s", (name or candidates), lib)
-            if sys.platform == 'win32':
-                loaded_lib = load_library(lib, name, win_cls)
-            else:
-                loaded_lib = load_library(lib, name, others_cls)
-        else:
-            _LOGGER.error('%r could not be found', (name or candidates))
-            raise LibraryNotFoundException(name)
+        liblist = locate_library(candidates, find_library)
+        for lib in liblist:
+            try:
+                if lib:
+                    _LOGGER.debug("%r found as %s", (name or candidates), lib)
+                    if sys.platform == 'win32':
+                        loaded_lib = load_library(lib, name, win_cls)
+                    else:
+                        loaded_lib = load_library(lib, name, others_cls)
+                else:
+                    _LOGGER.error('%r could not be found', (name or candidates))
+                    raise LibraryNotFoundException(name)
```

Comment:
It seems that the case where no library candidate is found isn't properly handled: in your current implementation, `locate_library` would return an empty list, and the whole loop body would never execute, making these lines unreachable.